lib/peeps.rb

```diff
@@ -3,22 +3,23 @@
 
 class Peeps
 
-  attr_reader :id, :message
+  attr_reader :id, :message, :time
 
-  def initialize(id:, message:)
+  def initialize(id:, message:, time:)
     @id = id
     @message = message
+    @time = time
   end
 
   def self.all
       result = DatabaseConnection.query("SELECT * from peep ORDER by id DESC;")
       result.map do |peep|
-      Peeps.new(id: peep['id'], message: peep['message'])
+      Peeps.new(id: peep['id'], message: peep['message'], time: peep['time'])
   end
 end
 
-  def self.create(message:)
-      result = DatabaseConnection.query("INSERT INTO peep (message) VALUES('#{message}') RETURNING id,message;")
-      Peeps.new(id: result[0]['id'], message: result[0]['message'])
+  def self.create(message:,time: Time.now)
```

Comment:
I used Postrges's default current_time in the table, but I think your approach to use ruby's Time.now is less headachy.